### DISCUSSION TITLE
The Brand new TitleBar OR Shell navigation?

### DISCUSSION THREAD

**Discussion by mdelgadov** [dotnet/maui#25811] [Q&A] [answered]
I just tested the brand new TitleBar and found that in Windows, you go either Shell OR TitleBar, not both.

Is that the way it has to be? Can we have it all? To ask is cheap, implementing, however...

Thanks to the comnunity

**Comment by stephenquan** [comment, marked as answer]
`TitleBar` looks interesting. Meanwhile, perhaps you can use `Shell.TitleView` and how to [Display views in the navigation bar](https://learn.microsoft.com/en-us/dotnet/maui/fundamentals/shell/pages?view=net-maui-8.0#display-views-in-the-navigation-bar)?
